src/components/Camera.tsx

```diff
@@ -0,0 +1,66 @@
+import { useRef, useEffect, useState } from "react";
+import "./Camera.css";
+
+export const Camera = () => {
+  const [error, setError] = useState<string | null>(null);
+  const [isEnabled, setIsEnabled] = useState(false);
+  const videoRef = useRef<HTMLVideoElement | null>(null);
+  const streamRef = useRef<MediaStream | null>(null);
```

Comment:
Stream is not used outside useEffect. Lets try to move it into the useEffect as variable.